service-registry/src/main/java/io/servicecomb/serviceregistry/api/Const.java

```diff
@@ -37,6 +37,15 @@ private Const() {
 
     public static final String V4_PREFIX = String.format("/v4/%s/registry", DOMAIN_NAME);
 
+    public static final String REGISTRY_VERSION;
+    static {
+      if (VERSION_V3.equals(CURRENT_VERSION)) {
+        REGISTRY_VERSION = "3.0.0"; // old is strange
+      } else {
+        REGISTRY_VERSION = "0.4.0";
```

Comment:
I think we just need keep tracking the API version instead of the service center version.

Reply:
I am going to change this to latest according to wujimin's suggestion